### PR TITLE
8337241: Shenandoah: Normalize include guards

### PR DESCRIPTION
**Pull request by earthling-amzn**
Some of the include guards are not like the others.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8337241](https://bugs.openjdk.org/browse/JDK-8337241): Shenandoah: Normalize include guards (**Bug** - P4)

### Reviewers
 * [Aleksey Shipilev](https://openjdk.org/census#shade) (@shipilev - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/20342/head:pull/20342` \
`$ git checkout pull/20342`

Update a local copy of the PR: \
`$ git checkout pull/20342` \
`$ git pull https://git.openjdk.org/jdk.git pull/20342/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 20342`

View PR using the GUI difftool: \
`$ git pr show -t 20342`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/20342.diff">https://git.openjdk.org/jdk/pull/20342.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/20342#issuecomment-2251559599)

### CODE REVIEW

File: src/hotspot/share/gc/shenandoah/shenandoahController.hpp [hunk 1]
```diff
@@ -102,4 +102,4 @@ class ShenandoahController: public ConcurrentGCThread {
   size_t get_gc_id();
   void update_gc_id();
 };
-#endif //LINUX_X86_64_SERVER_SLOWDEBUG_SHENANDOAHCONTROLLER_HPP
+#endif //SHARE_GC_SHENANDOAH_SHENANDOAHCONTROLLER_HPP
```
Comment (shipilev) [inline src/hotspot/share/gc/shenandoah/shenandoahController.hpp:105]: Should have a space after `//`?

**Review by shipilev — APPROVED** [review]
Looks fine, with a nit.

File: src/hotspot/share/gc/shenandoah/shenandoahController.hpp (hunk 1 above)
Comment (earthling-amzn) [reply src/hotspot/share/gc/shenandoah/shenandoahController.hpp]: Aye, good catch.

**Review by shipilev — APPROVED** [review]
Still good and trivial.

**Pull request closed without merging**